<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscarPaciente</class>
 <widget class="QMainWindow" name="BuscarPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar paciente</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>29</y>
      <width>1061</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pacientes</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablaPacientes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>1041</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Paterno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sexo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Peso</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estatura</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Edad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de sangre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alergias</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="botonBuscar">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>290</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar a Todos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdPaciente">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarId">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputNombre">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarNombre">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por apellido paterno</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAp">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>220</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarApellidoP">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
  <action name="actionID">
   <property name="text">
    <string>ID</string>
   </property>
  </action>
  <action name="actionNombre">
   <property name="text">
    <string>Nombre</string>
   </property>
  </action>
  <action name="actionTipo_de_sangre">
   <property name="text">
    <string>Tipo de sangre</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
